### PR TITLE
Replace bash with zsh

### PR DESCRIPTION
**Pull request by tspmelo**
Signed-off-by: Tiago Melo <tmelo@suse.com>

- new theme:
image
It shows the hosts, path, git on the left; and the return from previous command, if >0, in the right.
- new completion plugin for our scripts:
image

### CODE REVIEW

File: shared/zsh/dash.plugin.zsh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#autoload
+
+dash () {
+  $1.sh
+}
+
+_dash() {
+  local -a commands
+
+  commands=(
```
Comment (ricardoasmarques) [inline shared/zsh/dash.plugin.zsh]: Please add the recently merged `make-frontend.sh` command - https://github.com/ricardoasmarques/ceph-dev-docker/pull/36

Comment (tspmelo) [reply shared/zsh/dash.plugin.zsh]: Done

**Comment by tspmelo** [comment]
Added info about setup.sh in the README

File: Dockerfile [hunk 2]
```diff
@@ -25,12 +25,20 @@ RUN zypper -n in google-chrome-stable
 
 ENV CHROME_BIN /usr/bin/google-chrome
 
-ADD bin/ /root/bin/
-ADD bash.bashrc /etc/bash.bashrc
+#oh-my-zsh
+ENV ZSH_DISABLE_COMPFIX true
+RUN sh -c "$(wget https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh -O -)"
```
Comment (LenzGr) [inline Dockerfile]: oh-my-zsh FTW!

**Comment by tspmelo** [comment]
> @tspmelo can you please rebase this PR?

Done

**Pull request merged**